src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -197,7 +199,9 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
         return
 
     pr_comment = None
-    pr_comment_query = PullRequestComment.objects.filter(pull_request__id=pullrequest_id)
+    pr_comment_query = PullRequestComment.objects.filter(
+        pull_request__id=pullrequest_id, comment_type=CommentType.MERGED_PR
```

Comment:
```suggestion
        pull_request__id=pullrequest_id, comment_type=comment_type
```

Reply:
This is in the main workflow function which is specifically for merged prs. so comment_type can only be `MERGED_PR`

Reply:
oh i missed that. my bad!